go.mod

```diff
@@ -27,4 +28,7 @@ require (
 	nhooyr.io/websocket v1.7.4
 )
 
+// Temporary workaround to support ES256K signature alg (until https://github.com/square/go-jose/pull/278 got merged)
```

Comment:
This will be removed as soon as https://github.com/square/go-jose/pull/278 got merged.